src/assets/css/_sass/components/_projects-grid-item.scss

```diff
@@ -0,0 +1,20 @@
+.projects-grid-item {
+  flex-basis: calc((100% / 3) - 20px);
+  flex-grow: 0;
+  flex-shrink: 1;
+  max-width: 310px;
+  height: 440px;
+  margin-bottom: 20px;
+  text-decoration: none;
```

Comment:
sizing for product grid items; this is somewhat all over the place as far as sizing goes. I stole it from CanSwe products page.

Reply:
Yeah, this needs to be cleaned up, but for now is fine. Can you add a #todo comment to this?